internal/pkg/cli/deploy/lbws.go

```diff
@@ -151,7 +151,7 @@ func (d *lbWebSvcDeployer) stackConfiguration(in *StackRuntimeConfiguration) (*s
 		return nil, err
 	}
 	var opts []stack.LoadBalancedWebServiceOption
-	if !d.lbMft.NLBConfig.IsEmpty() {
+	if !d.lbMft.NLBConfig.PrimaryRoutingRule.IsEmpty() {
```

Comment:
Why do we differentiate primary routing route and additional routing rules?

Reply:
Because we have same routing rule details in both primary as well as additional routing rules, so it is efficient to separate out the common parameters between these two out and call them primary and additional.

Reply:
I meant why can't we combine primary routing rule and additional routing rule?

Reply:
I did not understand your question. How do we combine them? Today we define routing rule config inside `http` and after multiple port feature we will define more routing rules under  `http.additional_rules`. I am not sure combining them means what?